helm-charts/kube-prometheus/values.yaml

```diff
@@ -0,0 +1,4 @@
+############################################
+# NOTE: This file is managed by Terraform  #
+# Don't make any manual modification to it #
+############################################
```

Comment:
Probably worth mentioning that this is a placeholder file for `local_file` to overwrite.